.github/workflows/release.yml

```
@@ -65,28 +65,21 @@ jobs:
-      - name: Read release notes
-        id: release_notes
-        run: echo "::set-output name=notes::$(cat release_notes.md)"
+      - name: Rename APK
+        run: |
+          mv build/app/outputs/flutter-apk/app-prod-release.apk build/app/outputs/flutter-apk/mybudget-v${{ steps.get_version.outputs.version }}+${{ steps.get_version.outputs.build }}.apk
 
-      - name: Create GitHub Release
-        id: create_release
-        uses: actions/create-release@v1
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)